src/faldbt/project.py

```diff
@@ -379,11 +381,20 @@ def _model(
         target_model: MaybeNonSource = self._manifest.nativeManifest.resolve_ref(
             target_model_name, target_package_name, self.project_dir, self.project_dir
         )
-
         package_str = f"'{target_package_name}'." if target_package_name else ""
         model_str = f"{package_str}'{target_model_name}'"
         if target_model is None:
-            raise Exception(f"Could not find model {model_str}")
+            # Check if target_model_name is actually an alias
```

Comment:
So this enables doing `ref('agent_wait_time')` and `ref('wait_time')`?

Reply:
And is this the `dbt` behavior too? Yo can use the alias?

Reply:
Yes. Actually, if the table is not there, dbt creates a table with the alias instead of the model name, so if an alias is different from model name, our `ref(model_name)` stops working.

Reply:
"And in case it does, I would ask you to add a test for it"

@chamini2 does this mean testing alias feature of dbt in fal? Just an outline of a test would help clarify things.